frontend/src/scene/main-view/MainView.js

```diff
@@ -32,6 +34,17 @@ class MainView extends React.Component {
                     show={this.state.showCourseBalance}
                     onClear={() => this.setState({ showCourseBalance: false })}
                 />
+                <PaymentView
+                    show={
+                        this.props.location.pathname === '/payment-complete' ||
```

Comment:
location.pathname is meant to be read-only

Reply:
corrected